Template/Tag/EtrackerTag.web.js

```diff
@@ -44,7 +44,8 @@
                 if (etrackerConfig.etrackerDNT) {
                     script.setAttribute('data-respect-dnt', 'true');
                 }
-                script.src = '//static.etracker.com/code/e.js';
+                script.src = '//code.etracker.com/code/e.js';
+                script.setAttribute('async', '');
```

Comment:
@ben-dw If we change to async, we might have to adjust the `sendEvent` code too as per this [link](https://www.etracker.com/en/docs/faq-2/core-web-vitals/how-do-i-adjust-the-etracker-code-so-that-the-etracker-script-is-loaded-asynchronously/)

Reply:
@AltamashShaikh : Only ecommerce events have to be changed. I will double check within #311

Reply:
I have changed it in #311